README.md

```diff
@@ -1 +1,57 @@
-# to-do-list
\ No newline at end of file
+
+
+# To Do list: list structure
+
+> This is the first milestone of the To Do List project with the backbone or basic structure of the project.
+
+## Project screenshot
+
+![screenshot](./app_screenshot.png)
+
+Additional description about the project and its features.
+
+## Live Demo
+
+[Live Demo Link](https://mosams.github.io/to-do-list/dist/)
+
+## Built With
+
+- HTML
+- CSS
+- JAVASCRIPT
+
+## How to Setup
```

Comment:
The project looks great! Please add instructions to set the project up locally. It should include `npm install` and `npm start` commands.